### PR TITLE
HiddenResult: Improve help for contents parameter

### PR DESCRIPTION
**Pull request by troublemagnet**
Improves the docstring to explain the reason
behind existence of the parameter.

Closes https://github.com/coala/coala/issues/4672

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by troublemagnet** [comment]
@Makman2 All the tests have passed and rebased too :D

File: coalib/results/HiddenResult.py [hunk 1]
```diff
@@ -21,8 +21,8 @@ def __init__(self, origin, contents):
         obj.contents later.
 
         :param origin:   The originating bear.
-        :param contents: Any object that is picklable since it will be
-                         transferred across processes.
+        :param contents: Sends additional pickable data to the dependent
+                         bear besides a message.
```
Comment (Makman2) [inline coalib/results/HiddenResult.py]: Make the sentence passive:
```
Additional picklable data accessible by dependent bears besides ``message``.
```
\+
pickable -> pick**l**able :)

**Comment by Makman2** [comment]
Oh just seeing, you have modified `HiddenResult` instead of `Result`. `HiddenResult` is currently modified in another PR, where the `message` parameter gets included in `__init__`. You can modify `HiddenResult` too, but I would recommend doing that on a new PR and rebasing onto the mentioned existing PR :+1:

**Comment by Makman2** [comment]
Man what I'm talking about... sorry got confused.

**Comment by IpshitaC** [comment]
ack db592c34134df846f1006895279103d9c2eb2838

**Comment by Makman2** [comment]
Needs rebase and conflict resolution, but shouldn't be hard :+1: (after rebase the docstring actually does fit better because the `message` parameter does appear then in the function signature :D)

**Comment by stellargo** [comment]
ack db592c34134df846f1006895279103d9c2eb2838

**Comment by troublemagnet** [comment]
Closing the PR since the issue was resolved in #4757.

**Pull request closed without merging**